packages/client/src/utils/constants.ts

```diff
@@ -24,3 +24,7 @@ export const errors = {
 
 /* URLS */
 export const BASE_URL = 'https://ya-praktikum.tech/api/v2'
+
+
+/**for oAuth */
+export const REDIRECT_URL = 'http://localhost:3000';
```

Comment:
В задеплоеном приложении на localhost ссылаться бессмысленно. Можно ли сделать так?
```suggestion
export const REDIRECT_URL = '/';
```

Reply:
нет, там нужна с доменом  ссылка, так как по ней осуществляется не внутренний переход, а с яндекса.
но ты прав, плохо в константе это хранить. переделала на  window.location.origin . Насколько я знаю, если у сервиса несколько доменов, то они все регистрируются в oAuth яндекса.